exodus_core/analysis/static_analysis.py

```diff
@@ -325,9 +325,9 @@ def _get_icon_from_details(self, path):
         if details is not None:
             for i in details.get('images'):
                 if i.get('imageType') == 4:
-                    f = urllib.request.urlopen(i.get('url'))
+                    f = requests.get('url')
```

Comment:
I think there is something missing, should be `f = requests.get(i.get('url'))`

Reply:
I do not know. i do not think so.

Reply:
`requests.get('url')` won't work, because it will make a GET request on the string 'url'
The URL of the image here is in i.get('url')

Reply:
Ok I just changed it.